akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/groupchats/EditGroupChatMessageReqMsgHdlr.scala

```diff
@@ -52,18 +64,24 @@ trait EditGroupChatMessageReqMsgHdlr extends HandlerHelpers {
           val chatIsPrivate = groupChat.access == GroupChatAccess.PRIVATE
           val userIsAParticipant = groupChat.users.exists(u => u.id == user.intId)
           val userIsOwner = gcMessage.sender.id == user.intId
+          val isCustomPluginMessage = getIsCustomPluginMessage(gcMessage)
+
+          val allowEditing = userIsOwner && !isCustomPluginMessage
```

Comment:
I believe custom messages should be editable. 

What I'm against is allowing editing through the chat text area. Because plugin custom messages content can be structured in ways that don’t display properly in a plain textarea, editing should be done by the plugin itself, since it knows how to interpret and modify its own data.